_includes/head.html

```diff
@@ -0,0 +1,11 @@
+<head>
+    <meta charset="UTF-8">
+    <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="generator" content="Asciidoctor 1.5.2">
+    <title>Vaquero Docs</title>
```

Comment:
Needs to be `{{ page.title }}` and we may want to set a default in configuration if one is not set on the markdown. https://jekyllrb.com/docs/configuration/#front-matter-defaults

Reply:
Cool - good catch. Although, I couldn't figure out how to set the default in _config, I set it inline as I found here: https://github.com/jekyll/jekyll/issues/3046